pkg/phases/monitoring/monitoring.go

```diff
@@ -120,8 +149,7 @@ func Install(p *platform.Platform) error {
 			return fmt.Errorf("install: failed to apply CRD: %v", err)
 		}
 	}
-
-	return deployThanos(p)
```

Comment:
Was deployThanos duplicated or moved somehwere else?

Reply:
It is just how git formatted the patch, `deployThanos` is above a couple lines: https://github.com/flanksource/karina/pull/622/files#diff-4be420f64178e87f72203d316af8036bc200ab04f179f27f5f3db3bfa59952d9R121